tests/dummy/config/targets.js

```diff
@@ -2,13 +2,6 @@
 
 const browsers = ['last 1 Chrome versions', 'last 1 Firefox versions', 'last 1 Safari versions'];
 
-const isCI = Boolean(process.env.CI);
-const isProduction = process.env.EMBER_ENV === 'production';
-
-if (isCI || isProduction) {
-  browsers.push('ie 11');
-}
```

Comment:
Should we also remove IE11 from build targets for documentation app? https://github.com/kaliber5/ember-bootstrap/blob/master/docs/config/targets.js

Reply:
Done